<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>passenger</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pictures.qrc">
    <normaloff>:/newPrefix/index.jpeg</normaloff>:/newPrefix/index.jpeg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>691</width>
      <height>411</height>
     </rect>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>571</width>
       <height>341</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Request Trip</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>551</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Source : </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Galaxy : </string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLineEdit" name="source_planet"/>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="source_galaxy"/>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Planet : </string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>551</width>
         <height>231</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="1" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="add_dest">
            <property name="text">
             <string>Add Destination</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="remove_dest">
            <property name="text">
             <string>Remove </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Destination : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Galaxy : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="d_galaxy"/>
          </item>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Planet : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="d_planet"/>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Trip Type : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="VIP_trip">
            <property name="text">
             <string>VIP</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="regular_trip">
            <property name="text">
             <string>Regular</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="request_trip">
          <property name="text">
           <string>Request Trip</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>426</width>
             <height>136</height>
            </rect>
           </property>
           <widget class="QPlainTextEdit" name="Text">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>431</width>
              <height>141</height>
             </rect>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget_9">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>10</y>
         <width>391</width>
         <height>21</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="0">
         <widget class="QLabel" name="trip_distance">
          <property name="text">
           <string>Distance : </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="trip_cost">
          <property name="text">
           <string>Cost : </string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Manage Trip</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_7">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>40</y>
         <width>391</width>
         <height>161</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="1" column="2">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="Trip_status">
            <property name="text">
             <string>Trip Status</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cancel_trip">
            <property name="text">
             <string>Cancel Trip</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="driver_rate"/>
        </item>
        <item row="1" column="1">
         <widget class="QScrollArea" name="scrollArea_2">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>214</width>
             <height>125</height>
            </rect>
           </property>
           <widget class="QPlainTextEdit" name="status_text">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>241</width>
              <height>161</height>
             </rect>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="rate_driver">
          <property name="text">
           <string>Rate Driver</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Driver Rate : </string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Charge Account</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>80</y>
         <width>241</width>
         <height>91</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <widget class="QLineEdit" name="add_to_credit"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Add to credit :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="charge">
          <property name="text">
           <string>charge</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Discount Code</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_8">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>70</y>
         <width>381</width>
         <height>112</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="1" column="0">
         <widget class="QPushButton" name="getcode">
          <property name="text">
           <string>Get Discount Code</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QScrollArea" name="scrollArea_3">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>377</width>
             <height>76</height>
            </rect>
           </property>
           <widget class="QPlainTextEdit" name="code_text">
            <property name="geometry">
             <rect>
              <x>3</x>
              <y>0</y>
              <width>371</width>
              <height>78</height>
             </rect>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>571</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="Username_label">
        <property name="text">
         <string>User : </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="credit_label">
        <property name="text">
         <string>Credit : </string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="Date_Time_label">
        <property name="text">
         <string>Date_Time : </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>370</y>
       <width>88</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Logout</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>451</width>
      <height>271</height>
     </rect>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>381</width>
       <height>177</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLineEdit" name="register_username"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Phone Number :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Username :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="register_phonenumber"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Password :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="register_password"/>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="register_button">
        <property name="text">
         <string>Register</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="Back_register">
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Discount Code : </string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="register_discount_code"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>731</width>
      <height>301</height>
     </rect>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>331</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>login</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Username :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Password :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>register</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>749</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pictures.qrc"/>
 </resources>
 <connections/>
</ui>
